msmbuilder/tests/test_ligandfeaturizers.py

```diff
@@ -0,0 +1,104 @@
+import warnings
+
+import mdtraj as md
+import numpy as np
+
+from msmbuilder.featurizer import LigandContactFeaturizer
+from msmbuilder.featurizer import BinaryLigandContactFeaturizer
+from msmbuilder.featurizer import LigandRMSDFeaturizer
+
+
+def _random_trajs():
+    top = md.Topology()
```

Comment:
i really like this function, we should put this somewhere more accessible? Maybe mdtraj.utils should have this?